src/applications/caregivers/config/form.js

```diff
@@ -339,75 +397,103 @@ const formConfig = {
+              [secondaryCaregiverFields.secondaryOne.address]: address,
               [secondaryCaregiverFields.secondaryOne.primaryPhoneNumber]:
-                secondaryOneCaregiverProps.primaryPhoneNumber,
+                secondaryCaregiverOneProps.primaryPhoneNumber,
               [secondaryCaregiverFields.secondaryOne.alternativePhoneNumber]:
-                secondaryOneCaregiverProps.alternativePhoneNumber,
+                secondaryCaregiverOneProps.alternativePhoneNumber,
               [secondaryCaregiverFields.secondaryOne.email]:
-                secondaryOneCaregiverProps.email,
+                secondaryCaregiverOneProps.email,
+              [secondaryCaregiverFields.secondaryOne.verifyEmail]:
+                secondaryCaregiverOneProps.email,
               [secondaryCaregiverFields.secondaryOne.vetRelationship]:
-                secondaryOneCaregiverProps.vetRelationship,
+                secondaryCaregiverOneProps.vetRelationship,
               [secondaryCaregiverFields.secondaryOne
-                .hasSecondaryTwoCaregiverView]: {
+                .hassecondaryCaregiverTwoView]: {
                 type: 'boolean',
               },
             },
           },
         },
-        secondaryTwoCaregiverOne: {
+        secondaryCaregiverTwo: {
           path: 'secondaryTwo-caregiver-1',
-          title: 'Secondary Caregiver Information',
-          depends: formData => hasSecondaryTwoCaregiver(formData),
+          title: ' ',
+          depends: formData => hasSecondaryCaregiverTwo(formData),
           uiSchema: {
             'ui:description': SecondaryCaregiverInfo,
             // secondaryTwo UI
-            [secondaryCaregiverFields.secondaryTwo.fullName]:
-              secondaryCaregiverUI.secondaryTwo.fullNameUI,
-            [secondaryCaregiverFields.secondaryTwo.ssn]:
-              secondaryCaregiverUI.secondaryTwo.ssnUI,
-            [secondaryCaregiverFields.secondaryTwo.dateOfBirth]:
-              secondaryCaregiverUI.secondaryTwo.dateOfBirthUI,
-            [secondaryCaregiverFields.secondaryTwo.gender]:
-              secondaryCaregiverUI.secondaryTwo.genderUI,
-            [secondaryCaregiverFields.secondaryTwo.address]: addressUI,
+            [secondaryCaregiverFields.secondaryTwo.fullName]: fullNameUI(
+              'Secondary Two Family Caregiver',
+            ),
+            [secondaryCaregiverFields.secondaryTwo.ssn]: ssnUI(
+              'Secondary Two Family Caregiver',
+            ),
+            [secondaryCaregiverFields.secondaryTwo.dateOfBirth]: dateOfBirthUI(
+              'Secondary Two Family Caregiver',
+            ),
+            [secondaryCaregiverFields.secondaryTwo.gender]: genderUI(
+              'Secondary Two Family Caregiver',
+            ),
+            [secondaryCaregiverFields.secondaryTwo
+              .address]: addressWithoutCountryUI(
+              'Secondary Two Family Caregiver',
+            ),
           },
```

Comment:
LOL ok now I'm totally confused. <img alt="ghe-emoji" title=":_tuzki_help:" src="https://cloud.githubusercontent.com/assets/136959/13203633/f383f588-d881-11e5-8e50-14f5859d9e9f.gif">

Reply:
More silly stakeholder requirements if you are commenting about the "No country" part lol